mobile-config.js

```diff
@@ -2,12 +2,21 @@ App.info({
   id: 'org.codeforboston.mbta-ninja',
   name: 'MBTA Ninja',
   description: 'Crowdsource MBTA T alerts',
+  author: 'Code for Boston',
+  website: 'http://www.mbta.ninja/',
   version: '1.0.1'
 });
 
 App.icons({
+    // Android
     'android_ldpi': 'resources/icons/icon-ldpi.png',
     'android_mdpi': 'resources/icons/icon-mdpi.png',
     'android_hdpi': 'resources/icons/icon-hdpi.png',
-    'android_xhdpi': 'resources/icons/icon-xhdpi.png'
+    'android_xhdpi': 'resources/icons/icon-xhdpi.png',
+
+    // iOS
+    'iphone': 'public/apple-touch-icon-60x60.png',
+    'iphone_2x': 'public/apple-touch-icon-120x120.png',
+    'ipad': 'public/apple-touch-icon-57x57.png',
```

Comment:
Looks like https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/IconMatrix.html indicates iPad should be 76X76 and ipad_2x should be 152X152. Could you update please? I'll merge after!